core/src/test/java/org/apache/calcite/test/MaterializationTest.java

```diff
@@ -2389,6 +2389,19 @@ private void checkSatisfiable(RexNode e, String s) {
     }
   }
 
+  @Test public void testMaterializationAfterTrimingOfUnusedFields() {
+    String sql =
+        "select \"y\".\"deptno\", \"y\".\"name\", \"x\".\"sum_salary\"\n" +
```

Comment:
It seems that there are checkstyle violations in this method.
Please, before submitting a change, verify that the build passes locally using `mvn clean install`.